components/AddFundsBtn.js

```diff
@@ -5,7 +5,7 @@ import { FormattedMessage } from 'react-intl';
 import AddFundsModal from './host-dashboard/AddFundsModal';
 import StyledButton from './StyledButton';
 
-const AddFundsBtn = ({ children, collective, host }) => {
+const AddFundsBtn = ({ children, collective, host, isFromCollective }) => {
```

Comment:
While it's correct to show "Add funds from Collective" in the interface, from the technical point of view we're really adding funds from the **parent** (ie. we could allow organizations to host events in the future). So I think we should rename this `isFromCollective` in the code, to rather have something like `isFromParent` or `isFromParentAccount`.

Reply:
That makes sense. 👍🏽 I've renamed it to `isFromParent`. Let me know if everything looks good now.